### PR TITLE
Dynamic handlers

### PR DESCRIPTION
**Pull request by iilyak**

### CODE REVIEW

**Comment by kxepal** [comment]
`Add and` - you mean `Add mango_app`?

File: src/mango_httpd_handlers.erl [hunk 1]
```diff
@@ -0,0 +1,12 @@
+-module(mango_httpd_handlers).
```
Comment (kxepal) [inline src/mango_httpd_handlers.erl]: Missed license header.

File: src/mango_sup.erl [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+-export([start_link/1]).
+
+
+start_link(Args) ->
+    supervisor:start_link({local,?MODULE}, ?MODULE, Args).
+
+init([]) ->
+    {ok, {{one_for_one, 3, 10}, [
+        chttpd_handlers:registration_childspec(mango, mango_httpd_handlers)
```
Comment (kxepal) [inline src/mango_sup.erl]: There is no such module yet.

**Comment by rnewson** [comment]
I really like the shape of this. +1 to merge after squashing the license header in.

**Comment by iilyak** [comment]
PR is squashed

**Pull request merged**